mapscript/python/tests/cases/testing.py

```diff
@@ -41,22 +41,25 @@
 import sys
 import distutils.util
 import unittest
+import tempfile
 
 # define the path to mapserver test data
-test_path = os.path.abspath(os.path.join(__file__ ,"../../../../.."))
-TESTS_PATH = os.path.join(test_path, "tests")
+TESTS_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__ ),"data"))
+# TESTS_PATH = os.path.join(test_path, "tests")
```

Comment:
why not removing the commented code here and below ?

Reply:
Yes these should be cleaned up. I was going to pylint the whole test suite as a separate pull request.